controllers/monitoring.go

```diff
@@ -192,7 +192,7 @@ func (r *ClusterGroupUpgradeReconciler) processMonitoredObject(
 	safeName := utils.GetSafeResourceName(mcvName, clusterGroupUpgrade, utils.MaxObjectNameLength, 0)
 	mcv, err := utils.EnsureManagedClusterView(
 		ctx, r.Client, safeName, mcvName, clusterName, object.Kind+"."+strings.Split(object.APIVersion, "/")[0],
-		object.Name, *object.Namespace, clusterGroupUpgrade.Namespace+"-"+clusterGroupUpgrade.Name)
+		object.Name, *object.Namespace, clusterGroupUpgrade.Name, clusterGroupUpgrade.Namespace)
 	if err != nil {
 		return false, err
 	}
```

Comment:
As we are now reconciling based on MCV status updates, I think the requeue interval for the non-error cases could be increased to 1 min in order to reduce the number of unnecessary reconcilations. In scale lab, it was observed that it could take mins for an MCV to become ready.